cluster/v1/types.go

```diff
@@ -170,3 +170,44 @@ type ManagedClusterList struct {
 	// Items is a list of managed cluster.
 	Items []ManagedCluster `json:"items"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ManagedClusterSet is a set of managed clusters.
+type ManagedClusterSet struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Spec defines the attributes of the desired managed clusters
+	Spec ManagedClusterSetSpec `json:"spec"`
+}
+
+// ManagedClusterSetSpec describes the attributes of the desired managed clusters
+type ManagedClusterSetSpec struct {
+	// Clusters represents a set of managed clusters in the cluster set
+	Clusters ClusterSelector `json:"clusterSelector,omitempty"`
```

Comment:
I think the variable name here should better be ClusterSelector

Reply:
Updated.